omnibus/config/projects/installer.rb

```diff
@@ -4,16 +4,16 @@
 # Copyright 2016-present Datadog, Inc.
 require "./lib/ostools.rb"
 
-name 'updater'
-package_name 'datadog-updater'
+name 'installer'
+package_name 'datadog-installer'
 license "Apache-2.0"
 license_file "../LICENSE"
 
 third_party_licenses "../LICENSE-3rdparty.csv"
 
 homepage 'http://www.datadoghq.com'
 
-INSTALL_DIR = ENV['INSTALL_DIR'] || '/opt/datadog/updater'
+INSTALL_DIR = ENV['INSTALL_DIR'] || '/opt/datadog/installer'
```

Comment:
same

Reply:
rebased on top of main with your PR merged, should be good now